src/components/MonitorList.vue

```diff
@@ -19,21 +19,21 @@
 
             <router-link v-for="(item, index) in sortedMonitorList" :key="index" :to="monitorURL(item.id)" class="item" :class="{ 'disabled': ! item.active }">
                 <div class="row">
-                    <div class="col-6 col-md-8 small-padding" :class="{ 'monitorItem': $root.userHeartbeatBar == 'bottom' || $root.userHeartbeatBar == 'none' }">
+                    <div class="col-6 col-md-8 small-padding" :class="{ 'monitorItem': $root.userHeartbeatBar === 'bottom' || $root.userHeartbeatBar === 'none' }">
                         <div class="info">
                             <Uptime :monitor="item" type="24" :pill="true" />
-                            {{ item.name }}
+                            <span class="ms-1">{{ item.name }}</span>
                         </div>
                         <div class="tags">
                             <Tag v-for="tag in item.tags" :key="tag" :item="tag" :size="'sm'" />
                         </div>
                     </div>
-                    <div v-show="$root.userHeartbeatBar == 'normal'" :key="$root.userHeartbeatBar" class="col-6 col-md-4">
+                    <div v-show="$root.userHeartbeatBar === 'normal'" :key="$root.userHeartbeatBar" class="col-6 col-md-4 small-padding">
```

Comment:
Maybe due to this? the padding top/left look small now.

Reply:
no, I made the top and bottom padding the same
![image](https://user-images.githubusercontent.com/1710840/135629679-548b5579-da1b-4315-9e65-578cc4e79d3c.png)

and I removed this space below the health bar:
![image](https://user-images.githubusercontent.com/1710840/135629704-3c726e6a-16c8-4d7f-8069-071db02a7e94.png)


I'll change it so the total size of the monitor item stays the same

Reply:
fixed:
![image](https://user-images.githubusercontent.com/1710840/135630303-bb746d11-bf05-42a7-8355-e643fe948a7f.png)